src/settings/BatchGroupConfigurationSettings/utils.js

```diff
@@ -5,78 +5,42 @@ import {
   pickBy,
 } from 'lodash';
 
-export const isUploadURIValid = (value) => {
-  const REGEXP_URI = new RegExp('^$|([f][t][p])([s])?://.+$');
+const REGEXP_URI = new RegExp('^$|([f][t][p])([s])?://.+$');
+
+export const validateUploadURI = (value) => {
   const isTrue = REGEXP_URI.test(value);
 
   if (value === undefined || isTrue) return undefined;
 
   return <FormattedMessage id="ui-invoice.settings.batchGroupConfiguration.isUploadURIValid" />;
 };
 
-export const saveExportConfig = (formValues, mutator, selectedBatchGroupId, fetchExportConfig, showCallout) => {
-  const credentialsMethod = formValues.credentialsId ? 'PUT' : 'POST';
-
+export const saveExportConfig = (
+  { username, password, ...values },
+  { exportConfig, credentials, exportConfigId },
+  creds = {},
+) => {
+  const isNew = !values.id;
+  const httpMethod = isNew ? 'POST' : 'PUT';
   const config = {
-    id: formValues.id || null,
-    batchGroupId: selectedBatchGroupId,
-    startTime: formValues.startTime || null,
-    weekdays: keys(pickBy(formValues.weekdays)),
-    enableScheduledExport: formValues.enableScheduledExport,
-    format: formValues.format,
-    uploadURI: formValues.uploadURI,
-  };
-
-  const credentials = {
-    username: formValues.username,
-    password: formValues.password,
-    exportConfigId: formValues.id,
+    id: values.id || null,
```

Comment:
what does `|| null` ? is there some edge cases?

Reply:
Removed it